src/js/view/controls/components/timeslider.js

```diff
@@ -201,7 +201,7 @@ class TimeSlider extends Slider {
         var duration = this._model.get('duration');
         var position;
         if (duration === 0) {
-            this._api.play(reasonInteraction());
+            this._api.playToggle(reasonInteraction());
```

Comment:
This doesn't make sense. `play` was not being used as intended. See:

https://github.com/jwplayer/jwplayer/commit/fd47a76df76b94aba4350bec5df2a0fae7efea3d#diff-5b00ecc1e80bb1fce91c8111d655806eL137

and

https://github.com/jwplayer/jwplayer/commit/9b3985aa4fc3cf0110321522955c43ba2aa72fc6#diff-5b00ecc1e80bb1fce91c8111d655806eR88

I think the intent was to initiate playback (call `play(true)`) when the stream hasn't begun. We definitely do not want to call `video.pause()` here.